canonical/lxd-ui
# Add configuration of migration options and nesting on instances and profiles

## Done

- Added "migration" configuration options on instances and profiles
- Added nesting option in security section for instances and profiles

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @mas-who or @edlerd for access.
    - With a local copy of this branch, [build and run as described in the docs](../CONTRIBUTING.md#setting-up-for-development).
2. Perform the following QA steps:
    - create a profile and check the new migration configuration options as well ass security>nesting work well
    - edit a profile and check the new migration configuration options as well ass security>nesting work well
    - create an instance and check the new migration configuration options as well ass security>nesting work well
    - edit an instance and check the new migration configuration options as well ass security>nesting work well
    - create a vm and check the new migration configuration options as well ass security>nesting work well
    - edit a vm and check the new migration configuration options as well ass security>nesting work well

## Screenshots

![image](https://github.com/user-attachments/assets/5062b3d3-010f-4482-8dec-78621127f86d)
